final_project/airflow/dags/enrich_user_profiles.py

```diff
@@ -0,0 +1,48 @@
+
+import os
+from airflow.decorators import dag
+from airflow.decorators import task
+from datetime import datetime
+from google.cloud import bigquery
+
+
+os.environ["no_proxy"] = "*"
+
+
+@dag(
+    schedule=None,
+    start_date=datetime(2021, 1, 1),
+    end_date=None,
+    max_active_runs=1
+    )
+def enrich_user_profiles():
+    big_query = bigquery.Client(location='europe-west3')
+
+    @task
+    def create_gold_table():
+        create_sql = """
+        CREATE OR REPLACE TABLE gold.user_profiles_enriched
```

Comment:
This is dangerous. In a real project the schema should be static, so "create table" is prohibited in Airflow (but many people use it anyway).